pkg/builder/group/constraint_group_builder.go

```diff
@@ -20,7 +20,7 @@ type ConstraintInfoBuilder struct {
 func newConstraintBuilder(constraintType proto.GroupDTO_ConstraintType, constraintId string) *ConstraintInfoBuilder {
 
 	return &ConstraintInfoBuilder{
-		constraintName: constraintId,
+		constraintId: constraintId,
```

Comment:
Also  add field for constraintDisplayName: constraintId,

Reply:
Added for both `constraintDisplayName` and `constraintName`.